### DISCUSSION TITLE
Accessing open EBS provisioned PV in pods across different namespaces

### DISCUSSION THREAD

**Discussion by Devesh31** [openebs/openebs#3528] [General] [answered]
How can we access persistent volume provisioned dynamically by open EBS in pods across different namespaces?
Suppose we have two namespaces ‘ns1’ and ‘ns2’ in a cluster and if the pod within ns1 has a claim requesting a specific storage volume and then the cluster dynamically provisions a volume using open EBS for that PVC. So pod will access that PV and some data is stored within that PV.So PV is bound to PVC in namespace ‘ns1’. So is there any way to access that PV within a pod which is in namespace ‘ns2’.

**Comment by niladrih** [comment]
Hi @Devesh31, which OpenEBS storage engine is this for?

**Reply by avishnu** [reply, marked as answer]
@Devesh31 , an OpenEBS PV can be accessed by only 1 pod at a time. It is possible to delete the first pod's PVC (retaining the PV) and then launching a second pod in a different namespace and request for the same PV.
